doc/rst/developer/bestPractices/ContributorInfo.rst

```diff
@@ -134,16 +148,81 @@ pass testing for ``start_test -performance``).
 
 .. _Creating a Simple Test: https://github.com/chapel-lang/chapel/blob/master/doc/rst/developer/bestPractices/TestSystem.rst#creating-a-simple-test
 
+.. _Design
+
+Design
+------
+
+.. _Creating a design issue
+
+Creating a design issue
+~~~~~~~~~~~~~~~~~~~~~~~
+
+The design issue should aim to clearly and concisely present some design
+decisions. Here are some strategies to make a good design issue:
+
+- Summarize issue at the top, preferably include concise code example(s).
+- Try not to make initial issue too long in order to lower the barrier for
+  someone getting involved.
+- Clearly define the problem this proposal aims to solve.
+- It can be useful to consider alternative solutions to the problem and
+  describe the trade-offs among them.
+- A design issue should have two developers not involved in the effort to review.
+- To find reviewers, send an email to chapel-developers_.
+
+  * Use a prefix of ``[Design]`` in the subject header.
+  * Include a short summary of the topic, potentially including motivation and/or an example.
+  * Include a link to your issue.
+  * Ask for people to volunteer to be involved in design discussions.
+
+    .. code-block:: text
+
+        Subject: [Design] Add feature A to module M
 
-.. _Request feedback on your changes:
+        Hello,
 
-Request feedback on your changes
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+        I would like to add feature A to module M. This feature is motivated by
+        X, Y, and Z. For example, this would enable:
+
+          var result = M.A(args);
+
+        See the issue here: https://github.com/chapel-lang/chapel/issues/<number>
+
+        I need 2 developers to identify themselves as API reviewers for this design
+        issue.
+
+        Thanks,
+        Contributor
+
+.. _Leading a design discussion
+
+Leading a design discussion
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Many design choices require a consensus among community members.
+It can be helpful to reach a consensus on a design decision quicker by making
+an effort to lead that discussion.
+
+Here are some ways to progress the discussion:
+
+- Ask people involved what they need to know to make a decision
+- Summarize the different solutions that were brought up in the discussion and
+  list their pros and cons
+- Compare with other languages, libraries, previous work in area
+
+GitHub does not have nested thread comments, so it can be helpful to create
+new issues for design elements generating a lot of discussion.
```

Comment:
This can be confusing for someone who's new to this.

Maybe we can emphasize that such spin-off issues should be opened for topics that are not perfectly on-topic for the main design issue, but there is still some merit in discussing them.

Also, it happens that we reach some design consensus on a major topic, but during implementation we realize smaller issues that grants some discussion. Those should also go in separate issues for follow up rather than tacking them onto the main discussion issue.

Reply:
That sounds reasonable.